### PR TITLE
Add qin and ignorebadquality options to BBDuk single/paired process

### PR DESCRIPTION
**Pull request by JureZmrzlikar**
* [x] Update CHANGELOG.rst for each commit separately:
* [x] Bump the process version:
* [x] All inputs are used in process.

### CODE REVIEW

File: resolwe_bio/processes/reads_processing/bbduk.yml [hunk 1]
```diff
@@ -479,6 +496,9 @@
           qtrim={{ operations.quality_trim }} \
         {% endif %} \
         trimq={{ operations.trim_quality }} \
+        {% if operations.ignore_bad_quality %} \
```
Comment (jkokosar) [inline resolwe_bio/processes/reads_processing/bbduk.yml:499]: `qin={{ operations.quality_encoding_offset }} \` is missing for the ``bbduk-single`` process.

File: resolwe_bio/processes/reads_processing/bbduk.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@
       cores: 10
       memory: 8192
   data_name: '{{ reads|sample_name|default("?") }}'
-  version: 2.4.1
+  version: 2.5.1
```
Comment (jkokosar) [inline resolwe_bio/processes/reads_processing/bbduk.yml:19]: Should be ``2.5.0``

File: resolwe_bio/processes/reads_processing/bbduk.yml [hunk 3]
```diff
@@ -552,7 +572,7 @@
...
```
Comment (jkokosar) [inline resolwe_bio/processes/reads_processing/bbduk.yml:575]: Should be ``2.5.0``

File: resolwe_bio/processes/reads_processing/bbduk.yml (hunk 1 above)
Comment (JureZmrzlikar) [reply resolwe_bio/processes/reads_processing/bbduk.yml:500]: Nice catch, thanks!

**Review by jkokosar — APPROVED** [review]

**Pull request merged**